node/cn/gasprice/gasprice.go

```diff
@@ -216,20 +222,10 @@ func (oracle *Oracle) suggestTipCapUsingFeeHistory(ctx context.Context) (*big.In
 	head, _ := oracle.backend.HeaderByNumber(ctx, rpc.LatestBlockNumber)
 	headHash := head.Hash()
 
-	// If the latest gasprice is still available, return it.
-	oracle.cacheLock.RLock()
-	lastHead, lastPrice := oracle.lastHead, oracle.lastPrice
-	oracle.cacheLock.RUnlock()
-	if headHash == lastHead {
-		return new(big.Int).Set(lastPrice), nil
-	}
 	oracle.fetchLock.Lock()
 	defer oracle.fetchLock.Unlock()
-
 	// Try checking the cache again, maybe the last fetch fetched what we need
```

Comment:
This comment is now irrelevant

Reply:
Fixed in [407d41e](https://github.com/kaiachain/kaia/pull/44/commits/407d41e1329b657a20ac7a0383952dd51d89d42a)